CoEpi/ui/symptomReport/FollowUp/Breathless/BreathlessViewController.swift

```diff
@@ -69,20 +69,21 @@ class BreathlessViewController: UIViewController {
         
         tableView.rx.itemSelected
             .subscribe(onNext: {indexPath in
+                
                 if indexPath[1] == 0{
-                    print ("0")
+                    self.viewModel.onCauseSelected(cause: .leavingHouseOrDressing)
```

Comment:
Do you need to use self here?

Reply:
yes. without it you get "Reference to property 'viewModel' in closure requires explicit 'self.' to make capture semantics explicit"